apps/e2e/src/indexer-client/fullSanity.ts

```diff
@@ -337,31 +337,47 @@ async function fullSanity(context: RunContext) {
 
   prettyPrint('Fast Withdrawal Signature', fastWithdrawalSignature);
 
-  const updateLeaderboardRegistrationResult =
-    await client.updateLeaderboardRegistration({
-      contestId: 12,
-      subaccountName: subaccount.subaccountName,
-      subaccountOwner: subaccount.subaccountOwner,
-      updateRegistration: {
-        verifyingAddr: endpointAddr,
-        chainId,
-      },
-    });
+  try {
+    const updateLeaderboardRegistrationResult =
+      await client.updateLeaderboardRegistration({
+        contestId: 17,
+        subaccountName: subaccount.subaccountName,
+        subaccountOwner: subaccount.subaccountOwner,
+        updateRegistration: {
+          verifyingAddr: endpointAddr,
+          chainId,
+        },
+      });
 
-  prettyPrint(
-    'Update leaderboard registration result',
-    updateLeaderboardRegistrationResult,
-  );
+    prettyPrint(
+      'Update leaderboard registration result',
+      updateLeaderboardRegistrationResult,
+    );
+  } catch (e: any) {
```

Comment:
nit: I think `unknown` works slightly better here, but you might not be able to retrieve `e?.response?.data`. End of the day, not a huge deal